### ISSUE TITLE
UnhandledPromiseRejectionWarning: TypeError when "Save a file with stream timestamps for each match?" set to YES

### ISSUE THREAD

**Issue by paulsp** [MayorMonty/tm-obs-switcher#9]
macOS 12.5.1 - Apple M2
tm-obs-switcher v1.3.1

I am getting the error below  "Save a file with stream timestamps for each match?" set to "Yes" and scan switch does not happen

 ```
Will save match stream times to /var/folders/vq/kcv78q8x61x556cnzwzttv9r0000gn/T/tm_obs_switcher_2022-09-18T19:24:41.629Z_times.csv
(node:96983) UnhandledPromiseRejectionWarning: TypeError: undefined is not a function
    at /snapshot/tm-obs-switcher/out/main.js
    at step (/snapshot/tm-obs-switcher/out/main.js)
    at Object.next (/snapshot/tm-obs-switcher/out/main.js)
    at fulfilled (/snapshot/tm-obs-switcher/out/main.js)
(Use `tm-obs-switcher-macos --trace-warnings ...` to show where the warning was created)
(node:96983) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:96983) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```
```
paul@hummingbird Downloads % ls -l  /var/folders/vq/kcv78q8x61x556cnzwzttv9r0000gn/T/tm_obs_switcher_2022-09-18T19:24:41.629Z_times.csv
-rw-r--r--  1 paul  staff  0 Sep 18 15:24 /var/folders/vq/kcv78q8x61x556cnzwzttv9r0000gn/T/tm_obs_switcher_2022-09-18T19:24:41.629Z_times.csv
paul@hummingbird Downloads % 
```
The scene switching works as expected when "Save a file with stream timestamps for each match?" set to "N".

**Comment by MayorMonty** [comment]
I was encountering this issue when testing, I believe it had to do with a call to `String.prototypereplaceAll,` which was not included in the version of Node I packaged in the executables. This should be fixed in the upcoming patch released later today

**Comment by MayorMonty** [comment]
This is fixed in main. I am waiting on a contribution from @AlecH92 to merge into a new release.

**Comment by AlecH92** [comment]
> This is fixed in main. I am waiting on a contribution from @AlecH92 to merge into a new release.

Hey there, if you'd like to pull in my changes go ahead! I am still playing around to see if this can replace some of my AutoHotKey scripts - great libraries though, I hadn't thought of hooking in to the webserver.
